idl/response/foxsi_get_spectral_image.pro

```diff
@@ -0,0 +1,378 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;FUNCTION:      "foxsi_get_spectral_image"
```

Comment:
I thought this function just creates a default "test" dataset? If so I think it should say that in the function name.